packages/typespec-ts/test/modularUnit/scenarios/apiOperations/apiOperations.md

```diff
@@ -431,7 +431,7 @@ export async function downloadFile(
 }
 ```
 
-# should handle contentTypes has multiple form data in response
+# skip: should handle contentTypes has multiple form data with part array in response and pending tcgc https://github.com/Azure/typespec-azure/issues/3091
```

Comment:
This is not supported in TCGC https://github.com/Azure/typespec-azure/issues/3091.